web-server/src/content/DoraMetrics/DoraCards/ChangeFailureRateCard.tsx

```diff
@@ -80,6 +81,12 @@ export const ChangeFailureRateCard = () => {
         .incident_count
   );
 
+  const changeFailureRateCount = useCountUp(
+    changeFailureRateProps.count || 0,
+    1500,
```

Comment:
If you intend to use 1500 as the duration everywhere, and for now this hook isn't used anywhere else...

Why not just use 1500 as the default argument?

Reply:
fixed